index.js

```diff
@@ -0,0 +1,9 @@
+import app from './server/config/express';
+
+const port = process.env.PORT || 3000;
+
+const server = app.listen(port, () => {
+  console.log('API Server started and listening on port 3000');
```

Comment:
I think you can use `port` here since you've defined it as opposed to using `3000`